client/src/main/java/com/arcbees/beestore/client/application/ShoppingCartItem.java

```diff
@@ -57,4 +62,8 @@ public void setQuantity(int quantity) {
     public void addMore(int quantity) {
         this.quantity += quantity;
     }
+
+    public String getStorageKeyName() {
+        return productDto.getBrand().getValue() + "" + productDto.getProductType().name();
```

Comment:
`""`: I think there should be a space right? If not, that empty string does nothing.